go.mod

```diff
@@ -3,117 +3,125 @@ module github.com/castai/kvisor
 go 1.23.2
 
 require (
-	github.com/ClickHouse/clickhouse-go/v2 v2.23.2
-	github.com/aquasecurity/trivy v0.51.2
-	github.com/castai/image-analyzer v0.5.2
-	github.com/cenkalti/backoff/v4 v4.2.1
+	github.com/ClickHouse/clickhouse-go/v2 v2.30.0
+	github.com/aquasecurity/trivy v0.56.2
+	github.com/castai/image-analyzer v0.6.0
+	github.com/cenkalti/backoff/v4 v4.3.0
 	github.com/cespare/xxhash/v2 v2.3.0
-	github.com/cilium/cilium v1.15.4
+	github.com/cilium/cilium v1.17.0-pre.1
 	github.com/cilium/ebpf v0.16.0
-	github.com/containerd/containerd v1.7.18
+	github.com/containerd/containerd v1.7.23
+	github.com/containerd/containerd/api v1.7.19
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
-	github.com/elastic/go-freelru v0.13.0
-	github.com/fatih/color v1.17.0
+	github.com/elastic/go-freelru v0.14.0
+	github.com/fatih/color v1.18.0
 	github.com/florianl/go-conntrack v0.4.0
-	github.com/go-playground/validator/v10 v10.17.0
-	github.com/golang/glog v1.2.0
-	github.com/google/go-containerregistry v0.19.1
+	github.com/go-playground/validator/v10 v10.22.1
+	github.com/golang/glog v1.2.2
+	github.com/google/go-containerregistry v0.20.2
 	github.com/google/gopacket v1.1.19
 	github.com/google/uuid v1.6.0
-	github.com/grafana/pyroscope-go v1.1.1
+	github.com/grafana/pyroscope-go v1.2.0
 	github.com/hashicorp/golang-lru/v2 v2.0.7
 	github.com/iancoleman/strcase v0.3.0
 	github.com/json-iterator/go v1.1.12
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/labstack/echo/v4 v4.11.3
+	github.com/labstack/echo/v4 v4.12.0
 	github.com/magiconair/properties v1.8.7
 	github.com/onsi/ginkgo v1.16.5
-	github.com/prometheus/client_golang v1.19.1
-	github.com/redis/go-redis/v9 v9.4.0
-	github.com/samber/lo v1.39.0
+	github.com/prometheus/client_golang v1.20.5
+	github.com/redis/go-redis/v9 v9.7.0
+	github.com/samber/lo v1.47.0
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/cobra v1.8.1
 	github.com/spf13/pflag v1.0.6-0.20210604193023-d5e0c0615ace
 	github.com/spf13/viper v1.19.0
 	github.com/stretchr/testify v1.9.0
-	github.com/testcontainers/testcontainers-go v0.33.0
-	github.com/tklauser/go-sysconf v0.3.12
+	github.com/testcontainers/testcontainers-go v0.34.0
+	github.com/tklauser/go-sysconf v0.3.14
 	github.com/vishvananda/netns v0.0.4
 	go.uber.org/atomic v1.11.0
 	go.uber.org/goleak v1.3.0
-	golang.org/x/net v0.28.0
+	golang.org/x/net v0.30.0
 	golang.org/x/sync v0.8.0
-	golang.org/x/sys v0.25.0
-	golang.org/x/time v0.5.0
-	golang.stackrox.io/kube-linter v0.4.1-0.20221021125313-bd11843210d1
-	google.golang.org/grpc v1.64.1
-	google.golang.org/protobuf v1.34.2
+	golang.org/x/sys v0.26.0
+	golang.org/x/time v0.7.0
+	golang.stackrox.io/kube-linter v0.6.8
+	google.golang.org/grpc v1.67.1
```

Comment:
google.golang.org/grpc is not following semver, did you check if there are no breaking changes?

Reply:
There is a breaking change related to authenticating with TLS certificates, but since we are not using it, we should be fine, but good catch 👌 

Normally renovate should list the changelog though 🤔 I wonder why it is missing.